docs/dictionary/keyword/else-if.lcdoc

```diff
@@ -0,0 +1,35 @@
+Name: else if
+
+Type: keyword
+
+Syntax: else if
```

Comment:
I think the syntax should maybe be
```
else if <condition> then
   <statementList>
```

Reply:
and then copy the `condition:` and `statenentList:` parameter descriptions from `if.lcdoc` once that's done.